app/pb_kits/playbook/props/percentage.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Playbook
+  module Props
+    class Percentage < Playbook::Props::Base
+      def validate(value)
+        value.nil? ||
+          (value.is_a?(Numeric) &&
+          value <= 100 &&
+          value >= 0)
```

Comment:
Would it make sense to refactor this to:

```rb
...
value.nil? || numeric_in_range?(value)
...
def numeric_in_range?(value)
  (value.is_a?(Numeric) &&
          value <= 100 &&
          value >= 0)
end
```

for readability up-top?

Reply:
Yep! That looks good!